config.js

```diff
@@ -112,6 +112,8 @@ module.exports = (function() {
     config.github.owner = 'github-owner';
     config.github.repository = 'github-repository';
 
+    config.slack.requestSigningSecret = 'slack-super-signing-secret';
```

Comment:
Si je mute en supprimant cette ligne, les tests ajoutés passent toujours avec le message "This kind of interaction is not yet supported by Pix Bot."

Il me manque quelques chose à configurer en local ?

Reply:
Probablement parce que tu a un .env configuré comme il faut.